libs/components/modals/src/lib/modules/confirm/confirm-instance.ts

```diff
@@ -8,5 +8,18 @@ export class SkyConfirmInstance {
    * returns a `SkyConfirmCloseEventArgs` object with information about the button that
    * users select. It returns the `'cancel'` action when users press the <kbd>Escape</kbd> key.
    */
-  public closed = new EventEmitter<SkyConfirmCloseEventArgs>();
+  public get closed(): Observable<SkyConfirmCloseEventArgs> {
+    return this.#closed;
```

Comment:
We'll need a separate variable set to `asObservable()` returned here, otherwise the methods on the ReplaySubject will be available to the consumer:
```
(instance.closed as ReplaySubject<SkyConfirmCloseEventArgs>).next();
```

Reply:
I've updated to `return this.#closed.asObservable();`